src/main/java/frc/robot/components/Pigeon.java

```diff
@@ -0,0 +1,69 @@
+
+package frc.robot.components;
+
+import com.ctre.phoenix.motorcontrol.can.TalonSRX;
+import com.ctre.phoenix.sensors.PigeonIMU;
+import edu.wpi.first.wpilibj.interfaces.Gyro;
+
+/**
+ * this class extends the PigeonIMU class for convenient API that implements Gyro and PIDSource.
+ */
```

Comment:
* this class extends the PigeonIMU class for convenient API that implements Gyro and PIDSource.
This class...